components/SingleWordHeading/SingleWordHeading.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+export default function SingleWordHeading({ base, query1 }) {
+  return (
+    <StyledHeadingWrapper>
+      <h2>{base.flag}</h2>
+      <h2>{query1.flag}</h2>
+      <h2>{base.translation}</h2>
+      <h2>
+        {query1.translation} {query1.gender && `(${query1.gender})`}
+      </h2>
+    </StyledHeadingWrapper>
+  );
+}
+
+const StyledHeadingWrapper = styled.section`
+  margin: 0.625rem 0.625rem 1.25rem 0.625rem;
+  background-color: White;
+  color: darkmagenta;
```

Comment:
a very cool new component 👍 
just an idea - nothing you have to change. Maybe you can place your colors to the globalStyles file. Than you can reference to it - e.g. "var(--darkmagenta)"